src/akamai/js/akamaiHD_flash.js

```diff
@@ -595,14 +595,16 @@ require("../../../html5-common/js/utils/constants.js");
     var raiseBitratesAvailable = function(event) {
       var vtcBitrates = [{id: "auto", width: 0, height: 0, bitrate: 0 }];
       if (event) {
-        for (var i = 0; i < event.eventObject.length; i++) {
-          var vtcBitrate = {
-            id: event.eventObject[i].id,
-            width: event.eventObject[i].width,
-            height: event.eventObject[i].height,
-            bitrate: event.eventObject[i].bitrate
+        for(var i in event.eventObject) {
```

Comment:
For the purpose of clarity, can we use another variable name here in place of `i`. The use of `i` is to denote index - as it previously did in the code. Since the intention is to iterate over properties, perhaps `prop` or `key` is more appropriate.